netbout-web/src/main/java/com/netbout/rest/account/TkIndex.java

```diff
@@ -62,7 +63,7 @@ public Response act(final Request req) throws IOException {
         return new RsPage(
             "/xsl/account.xsl",
             this.base,
-            req,
+            new RqWithDefaultHeader(req, "accept", "text/xml"),
```

Comment:
@exper0 take it easy :)
I'm sure you will come to an agreement
to be honest I shoudn't comment here before @pinaf accepted or rejected the pull request.
@pinaf is high skilled developer. I think he will explain his comments in details or take into account your arguments.
you, @pinaf and me work together to find the best decision